.vscode/spellchecker.json

```diff
@@ -0,0 +1,45 @@
+{
+    "ignoreWordsList": [
+        "Brockenhurst",
+        "Basingstoke",
+        "Gordano",
+        "Guildford",
+        "Headington",
+        "Markt",
+        "Indersdorf",
+        "Mirfield",
+        "Tunbridge",
+        "prepend",
+        "baseurl",
+        "blueshirt",
+        "blueshirts",
+        "Kickstart",
+        "Kickstarts",
+        "KickStart",
```

Comment:
We used to use "KickStart" quite heavily. I thought we'd removed all the usages of it, but there still seem to be some. Arguably we should not allow it since the chances of people editing the old files is quite low.